src/pages/home/HeaderView.js

```diff
@@ -1,89 +1,90 @@
 import React from 'react';
-import {View, Image, TouchableOpacity} from 'react-native';
+import {View, Image, Pressable} from 'react-native';
 import PropTypes from 'prop-types';
 import {withOnyx} from 'react-native-onyx';
 import Header from '../../components/Header';
 import styles from '../../styles/styles';
 import ONYXKEYS from '../../ONYXKEYS';
 import {withRouter} from '../../libs/Router';
-import LHNToggle from '../../../assets/images/icon-menu-toggle.png';
+import BackArrow from '../../../assets/images/chevron-left.png';
 import {PinIcon} from '../../components/Expensicons';
 import compose from '../../libs/compose';
 import {togglePinnedState} from '../../libs/actions/Report';
 
 const propTypes = {
-    // Toggles the hamburger menu open and closed
-    onHamburgerButtonClicked: PropTypes.func.isRequired,
+  // Toggles the navigationMenu open and closed
```

Comment:
These issues are because your IDE is set to indent with 2 spaces. We use 4 spaces. Set your IDE to use 4 spaces and convert these indentations.

Reply:
Add 4 Space in IDE.

Reply:
No tabs.